project/SparkBuild.scala

```diff
@@ -254,6 +256,12 @@ object SparkBuild extends PomBuild {
     }
   )
 
+  lazy val cachePolicies = if (sys.env.contains("SKIP_LOCAL_M2")) {
```

Comment:
If this is necessary, this work should be completed in a separate PR first.

Reply:
Okay, Let me try this solution first 😄